### ISSUE TITLE
The scrcpy display windo could be very good if it had frame. 

### ISSUE THREAD

**Issue by bereket-09** [Genymobile/scrcpy#1621]
Hello guy first I want to say that this is a very nice app you have here, I think i might have an idea about it too I don't know if you have been requested but this app would be very nice if it had some sort of frame like this screenshot, and I wanted to say again thank you

![Screenshot (618)](https://user-images.githubusercontent.com/64479153/88483395-4179a780-cf70-11ea-99b1-8963526d632a.png)

**Comment by brunoais** [comment]
I'm ok with it for as long as it's optional. I rather have that space used for more screen space.

**Comment by bereket-09** [comment]
yes, that's right but still having the frame on your screen will look
much better but as you said it should be optional because there are also
peoples that don't like it.
